### PR TITLE
Update _forms.less

### PR DESCRIPTION
**Pull request by coolcat-creations**
Fix for issue #20869

Pull Request for Issue #20869.

### Summary of Changes
Adding width:auto for .field-media-input

### Testing Instructions
Compile the template less files and test if it solves Issue #20869

### CODE REVIEW

**Comment by wojsmol** [comment]
@coolcat-creations This for 4.0-dev or staging?

**Comment by coolcat-creations** [comment]
Its for staging, sorry - I don't know why it got tagged with 4.0

**Comment by brianteeman** [comment]
Sorry my mistake
could you please run generatecss.php and update this PR with the updated css files

File: administrator/templates/isis/less/blocks/_forms.less [hunk 1]
```diff
@@ -350,3 +350,8 @@ textarea.noResize {
 .form-horizontal .controls > .radio.btn-group-yesno:first-child {
 	padding-top: 2px;
 }
+
+/* Fix field media width */
+.field-media-input {
+	width:auto;
```
Comment (Quy) [inline administrator/templates/isis/less/blocks/_forms.less]: Add space after `:`

**Comment by coolcat-creations** [comment]
I understood that everyone has to run its own compiler and composer and docker things on local since some weeks to test PRs? Is it still so that generatecss.php has to be run?

**Comment by brianteeman** [comment]
You are getting confused with J4 - even then we are currently still comitting the css

**Comment by webnet-assmann** [comment]
I have tested this item :white_check_mark: successfully on b22fc653305d4f84b1dae7b4cc8b8b99a114be37<br /><br />Successfully testet<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21786">issues.joomla.org/tracker/joomla-cms/21786</a>.</sub>

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on b22fc653305d4f84b1dae7b4cc8b8b99a114be37<br /><br />Tested this PR successfully.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21786">issues.joomla.org/tracker/joomla-cms/21786</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Pull request merged**